src/api/include/migraphx/migraphx.h

```diff
@@ -26,6 +26,9 @@
 
 #include <stdlib.h>
 #include <stdbool.h>
+
+#include <migraphx/c/export.h>
```

Comment:
I think I would prefer the header to be under `migraphx/api/export.h`.

Reply:
Done